### PR TITLE
Send notifications only when the same alert has been issued within a predefined period.

### PR DESCRIPTION
**Pull request by wooparadog**

### CODE REVIEW

File: alerter/alerter.go [hunk 1]
```diff
@@ -149,13 +154,48 @@ func (al *Alerter) incrAlertNum(m *models.Metric) {
...
+	al.lock.RLock()
+	defer al.lock.RUnlock()
+	v, ok := al.alertRecords.Get(m.Name)
+	if !ok {
+		return true
+	}
+	alerted := 0
+	for _, timeStamp := range v.([]uint32) {
+		if timeStamp > 0 && m.Stamp-timeStamp < 10 {
+			alerted += 1
```
Comment (hit9) [inline alerter/alerter.go]: Here lint

Comment (hit9) [inline alerter/alerter.go]: 10? cfg.Interval?

File: alerter/alerter.go [hunk 2]
```diff
@@ -149,13 +154,48 @@ func (al *Alerter) incrAlertNum(m *models.Metric) {
...
+	var records []uint32
+	al.lock.Lock()
+	defer al.lock.Unlock()
+	v, ok := al.alertRecords.Get(m.Name)
+	if ok {
+		records = v.([]uint32)
+	} else {
+		records = make([]uint32, 10)
+	}
+	if len(records) >= 10 {
```
Comment (hit9) [inline alerter/alerter.go]: 10? 走个cfg?

**Comment by hit9** [comment]
需求是OK的.

File: alerter/alerter.go (hunk 2 above)
Comment (wooparadog) [inline alerter/alerter.go]: Fixed

Comment (wooparadog) [inline alerter/alerter.go]: 哦, 还没有.

**Pull request merged**